replication/binlogsyncer.go

```diff
@@ -25,10 +26,12 @@ type BinlogSyncer struct {
 	c        *client.Conn
 	serverID uint32
 
-	host     string
-	port     uint16
-	user     string
-	password string
+	// LocalHost is the name of you want to present to the MySQL master. If it is not set it will default to os.Hostname()
+	LocalHost string
```

Comment:
We can change that. I was going to leave this as public, so that people can set this if they wanted.  If they don't set it, it would default to os.Hostname(). If they set it, that's what we would send.

Here is where the check is done.
https://github.com/gdey/go-mysql/blob/2818d7e7c262a2307607238300e79c9033a3d027/replication/binlogsyncer.go#L68-L75